pages/inspector/inspector-issues-view.mdx

```diff
@@ -121,48 +191,33 @@ From there, the creator of each saved view and admins can archive saved views by
-
-### Event info
-The event impacted by the issue, its volume and whether it is documented in your Avo tracking plan.
-- If the event is present in the tracking plan, you can navigate to it to review and/or update the event definition
-- If the event is missing from your tracking plan, you can add it to your tracking plan.
-
-### Property info
-The property impacted by the issue, and whether it is documented to be sent with the event in your tracking plan.
-- If the property is present on the event in your tracking plan, you can navigate to it to review and/or update.
-- If the property is present in your tracking plan but not set to be sent with your event, you can add it to the event definition.
-- If the property is missing from your tracking plan, you can add it to your tracking plan.
-
-<Callout type="info">Not all issues are related to properties. See [full list of issue types here](/explore-tracking-plan/issue-types-in-inspector).</Callout>
-
-![Image shows example issue details for an unexpected property.](/images/inspector/inspector-issue-details.png)
 
-### Issue across sources
-
-At the bottom of your issue details, you will see a list of all sources that are impacted by the issue. For each source you will see:
-Event volume, issue volume and issue percentage and a link to the issue details on the other sources.
-
-![An example iOS issue that also impacts Android](/images/inspector/inspector-issue-details-sources.png)
 
 ## Fixing an issue found in Inspector
+
 When you find an issue in Inspector, the problem can either be on the implementation or tracking plan side:
 If the implementation is not what it should be according to the tracking plan, we recommend copying the issue link, creating a task in your task management system and assigning it to the relevant team to look into and fix in code
 If the tracking plan is not representing how the event should be, and the implementation is correct, we recommend updating the tracking plan accordingly. In the issue details, you can add events and properties with a click of a button and view the tracking plan definition in the context you are in
 
-
-<Callout type="info">We are very excited about streamlining the process of fixing issues and are going to be working on that soon. Please [reach out](/help/troubleshooting) if you have any thoughts on how you would like to act on issues and what would make your team more efficient</Callout>
+<Callout type="info">
+  We are very excited about streamlining the process of fixing issues and are
+  going to be working on that soon. Please [reach out](/help/troubleshooting) if
+  you have any thoughts on how you would like to act on issues and what would
+  make your team more efficient
+</Callout>
 
 ## Why are there no issues in my issues view?
```

Comment:
Should this section just be a Troubleshooting section of its own?

Reply:
Sold! 112baf24db841b0a5f27f1a62b829d2921e3f52f